website/templates/emails/reviews_submission_status.txt.mako

```diff
@@ -0,0 +1,25 @@
+Hello ${user.fullname},
+
+% if is_pre_moderation:
+    Your submission "${reviewable_title}", submitted to ${provider.name} has ${'not been accepted. You may edit the '+ provider_preprint_word+ ' and resubmit, at which time it will becoming pending moderation.' if is_rejected else 'been accepted by the moderator and is now discoverable to others.'}
```

Comment:
Nitpick: This (and the other similar lines) would be easier to read with block `% if`s instead of inline. Also, could check `reviewable.reviews_state` instead of requiring `is_rejected` in the context.